src/library-authoring/empty-page/EmptyPage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const EmptyPageWrapper = ({ children }) => (
+  <div className="empty-sheet-wrapper">
+    <div className="empty-content">{children}</div>
+  </div>
+);
+
+EmptyPageWrapper.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const EmptyPage = ({
+  heading,
+  body,
+  children,
+}) => (
+  <EmptyPageWrapper>
+    <h3 className="empty-heading-text">{heading}</h3>
```

Comment:
Why you don't use paragon classes for reducing lots of additional styles?

Reply:
@ihor-romaniuk  change to .h3, this component need to update within the task of style refactoring